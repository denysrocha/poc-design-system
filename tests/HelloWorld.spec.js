/* eslint-disable */
import { shallowMount } from "@vue/test-utils";
import HelloWorld from "@/components/HelloWorld.vue";

describe("HelloWorld.vue", () => {
  it('renders HelloWorld with correct HTML', () => {
    const msg = "new message";
    const wrapper = shallowMount(HelloWorld, {
      propsData: { msg },
    });

    expect(wrapper.html()).toMatchSnapshot();
  });

  it("renders props.msg when passed", () => {
    const msg = "new message";
    const wrapper = shallowMount(HelloWorld, {
      propsData: { msg },
    });
    expect(wrapper.text()).toMatch(msg);
  });
});
