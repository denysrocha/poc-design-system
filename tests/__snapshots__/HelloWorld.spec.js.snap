// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HelloWorld.vue renders HelloWorld with correct HTML 1`] = `
"<div class="greetings">
  <h1 class="green">new message</h1>
  <h3> Youâ€™ve successfully created a project with <a href="https://vitejs.dev/" target="_blank" rel="noopener">Vite</a> + <a href="https://vuejs.org/" target="_blank" rel="noopener">Vue 3</a>. </h3>
</div>"
`;
